pkg/controllers/inventory/inventory.go

```diff
@@ -29,27 +29,54 @@ import (
 	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
 	"antrea.io/nephe/pkg/controllers/inventory/common"
 	"antrea.io/nephe/pkg/controllers/inventory/store"
+	"antrea.io/nephe/pkg/controllers/utils"
 	"antrea.io/nephe/pkg/logging"
 )
 
-type Inventory struct {
+type Inventory interface {
```

Comment:
Call this InventoryInterface and change InventoryImpl to Inventory.. @shenmo3

Reply:
Done